includes/admin/class-wcs-admin-meta-boxes.php

```diff
@@ -310,14 +318,14 @@ private static function can_renewal_order_be_retried( $order ) {
 
 		$can_be_retried = false;
 
-		if ( wcs_order_contains_renewal( $order ) && $order->needs_payment() && '' != wcs_get_objects_property( $order, 'payment_method' ) ) {
+		if ( wcs_order_contains_renewal( $order ) && $order->needs_payment() && '' != wcs_get_objects_property( $order, 'payment_method' ) ) { // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison
```

Comment:
I was like 'why not just change `!=` to `!==`.

Then I tried:
```
var_dump('' != 0);
var_dump('' !== 0);
```

and it prints
```
bool(false)
bool(true)
```

Not worth the risk of rewriting that.

Reply:
Thanks @shendy-a8c. Yeh, I wanted to avoid changing any behaviour.

I think this PHPCS rule helps improve code quality for future commits, but I agree that there is not much benefit in rewriting cases where we risk breaking the existing logic.